presenter/src/locales/en.json

```
@@ -40,8 +40,8 @@
       "sitemap": "Sitemap"
     },
     "home": "Home",
-    "humanAndProjects": "Human and Projects",
-    "instagramm": "Instagramm",
+    "humansAndProjects": "Humasn and Projects",
```

```suggestion
    "humanAndProjects": "Human and Projects",
```
Human hat keinen Plural - das kommt aus dem Design von Hardi und sollte korrekt sein.